File: Programmers/모의고사.py
def solution(answers):
    answer = []
    one = [1, 2, 3, 4, 5]
    two = [2, 1, 2, 3, 2, 4, 2, 5]
    three = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5]
    answer_count = [0, 0, 0]
    for i in range(len(answers)):
        if one[i % len(one)] == answers[i]:
            answer_count[0] += 1
        if two[i % len(two)] == answers[i]:
            answer_count[1] += 1
        if three[i % len(three)] == answers[i]:
            answer_count[2] += 1
    MAX = max(answer_count)
    for i in range(len(answer_count)):
        if answer_count[i] == MAX:
            answer.append(i + 1)

    return answer

File: Programmers JS/모의고사.js
function solution(answers) {
  var answer = [];
  let one = [1, 2, 3, 4, 5];
  let two = [2, 1, 2, 3, 2, 4, 2, 5];
  let three = [3, 3, 1, 1, 2, 2, 4, 4, 5, 5];
  let count = [0, 0, 0];

  for (let i = 0; i < answers.length; i++) {
    if (one[i % one.length] === answers[i]) {
      count[0]++;
    }
    if (two[i % two.length] === answers[i]) {
      count[1]++;
    }
    if (three[i % three.length] === answers[i]) {
      count[2]++;
    }
  }
  count.forEach((el, idx) => {
    if (el === Math.max(...count)) {
      answer.push(idx + 1);
    }
  });

  return answer;
}
